gobblin-modules/gobblin-kafka-09/resource/job-props/SampleStreamingJobConfig.pull

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
no - this is a sample pull file so end users know how to set it up - where would I move this?